CMakeLists.txt

```diff
@@ -65,6 +65,7 @@ add_subdirectory(clapack-3.2.1-CMAKE)
 add_subdirectory(CBLAS)
 add_subdirectory(lapacke)
 add_subdirectory(src)
+add_subdirectory(plover)
```

Comment:
maybe plover should be a subdirectory of src, seeing as it contains source code? but don't feel strongly about this, also see the benefit of keeping it separate